pkg/prm/config.go

```diff
@@ -44,6 +48,9 @@ func (p *Prm) GenerateDefaultCfg() {
 	}
 	log.Trace().Msgf("Setting default toolpath (%s: %s)", ToolPathCfgKey, defaultToolPath)
 	viper.SetDefault(ToolPathCfgKey, defaultToolPath)
+
+	log.Trace().Msgf("Setting default Timeout (%s: %s)", ToolTimeoutCfgKey, DefaultToolTimeout)
```

Comment:
For this I think we want the timeout interpolator to be `%d` instead of `%s` - a _digit_ instead of a _string_, as the value we're passing is an integer.

```suggestion
	log.Trace().Msgf("Setting default Timeout (%s: %d)", ToolTimeoutCfgKey, DefaultToolTimeout)
```

Reply:
Done